pkg/restic/commands.go

```diff
@@ -581,3 +588,11 @@ func (w *ResticWrapper) appendInsecureTLSFlag(args []interface{}) []interface{}
 	}
 	return args
 }
+
+func (c *Command) getCommandArgsAsString() string {
+	var cmdParts []string
+	for _, arg := range c.Args {
+		cmdParts = append(cmdParts, fmt.Sprintf("%v", arg))
```

Comment:
Are we going to use double quotation or single quotation?

Reply:
Double quotation